ts/sagas/backendStatus.ts

```diff
@@ -49,8 +49,9 @@ export function* backendStatusWatcherLoop(
       backendStatusSaga,
       getStatus
     );
-    const currentState: ReturnType<typeof backendServicesStatusSelector> =
-      yield* select(backendServicesStatusSelector);
+    const currentState: ReturnType<typeof backendInfoSelector> = yield* select(
```

Comment:
See comment on the reducer's file about removing this one in favour of two distinct selectors